src/App.js

```diff
@@ -1,9 +1,21 @@
+import AboutMe from "./components/AboutMe";
+import Navbar from "./components/Navbar";
 import Portfolio from "./components/Portfolio";
+import Contact from "./components/Contact";
+import { Route, Routes } from "react-router-dom";
```

Comment:
Try to import all of your external packages (like this one) at the top of the file, followed by internal imports and finally, any styles as this helps other developers to understand your code quicker.